src/cli-storage.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "node:fs";
+import os from "node:os";
+
+const __confDir = os.homedir() + "/.baseplate";
+if (!fs.existsSync(__confDir)) {
+  fs.mkdirSync(__confDir);
+}
+
+export async function storageProvider() {
+  // @ts-ignore
+  const conf = await import("conf");
+  return new conf.default({
```

Comment:
Is it ok for this function to be executed multiple times? I see that `await storageProvider()` is called a couple times. Would it be better to have a shared instance of conf rather than calling `new conf.default()` every time?

Reply:
since this is a CLI and application lifespan is short(it lives from start to end of executing a command) it wont make much difference, as storageProvider() should be called only once in each command, but I make this change to be on the safer side.